packages/web/examples/DateRangeControlled/src/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import moment from 'moment';
+
+import {
+	ReactiveBase,
+	DateRange,
+	ResultCard,
+	SelectedFilters,
+	ReactiveList,
+} from '@appbaseio/reactivesearch';
+
+import './index.css';
+
+class Main extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
+	}
+
+	dateQuery(value) {
+		let query = null;
+		if (value) {
+			query = [
+				{
+					range: {
+						date_from: {
+							gte: moment(value.start).format('YYYYMMDD'),
+						},
+					},
+				},
+				{
+					range: {
+						date_to: {
+							lte: moment(value.end).format('YYYYMMDD'),
+						},
+					},
+				},
+			];
+		}
+		return query ? { query: { bool: { must: query } } } : null;
+	}
+
+	render() {
+		return (
+			<ReactiveBase
+				app="airbeds-test-app"
+				credentials="X8RsOu0Lp:9b4fe1a4-58c6-4089-a042-505d86d9da30"
+				type="listing"
+			>
+				<div className="row">
+					<div className="col">
+						<DateRange
+							value={this.state.date}
+							onChange={(value) => {
+								this.setState({
+									date: {
```

Comment:
@ShahAnuj2610 I don't think that this example is useful or test anything. You're setting the same value which is already in the current state of `DateRange`.

Reply:
@bietkul have updated the example